obspy/core/inventory/response.py

```diff
@@ -961,6 +961,57 @@ def is_close(a, b):
 
         return sampling_rates
 
+    @classmethod
+    def get_flat_response(cls, sensitivity=1., frequency=1.,
```

Comment:
Do we want a default value for the sensitivity? I can see arguments for and against it but I just wanted to quickly discuss it.

Reply:
I am happy with either way.  Let me know and I can change it.